packages/mobile/src/components/enable-push-notifications-drawer/EnablePushNotificationsDrawer.tsx

```diff
@@ -175,11 +174,10 @@ const EnablePushNotificationsDrawer = () => {
         <Button
           title={messages.enable}
           onPress={enablePushNotifications}
-          containerStyle={styles.button}
+          style={styles.button}
```

Comment:
up to you on if we do this this pr, but ive seen a lot of design systems have a `fullWidth` prop, that adds `width: 100%`

Reply:
Let's discuss this more! I can see an argument against having `fullWidth` as a prop too

Reply:
Nvm, I went ahead and added it bc I realized there are 2 places where width 100 needs to be set